akka-actor-testkit-typed/src/main/mima-filters/2.6.20.backwards.exclude/rename-fishing-outcomes-continue-java.excludes

```diff
@@ -0,0 +1,2 @@
+# FishingOutcomes.continue() cannot be used in Java code due to reserved word, see https://github.com/akka/akka/issues/29951
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.testkit.typed.javadsl.FishingOutcomes.continue")
```

Comment:
Wrong directory, should be 2.6.19 as in binary changes against latest released version.

Reply:
Thank you @johanandren . I thought about new release :) . I squashed commits for better history.